### DISCUSSION TITLE
[Feature Request]: Choose "Work Shifts" using time picker

### DISCUSSION THREAD

**Discussion by pwSteve** [grafana/grafana#38313] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I would like to add a new feature for the time picker on the dashboards. Here it would be nice to select shifts and to switch between different shifts. For example "Morning", "Afternoon" and "Night". The dashboard should show the time range of the shift when you select it. With the arrow you can go back to see previous shifts.

**Why is this needed**:

In many factories you work in shifts. For those companies it is interesting to switch between the shifts to see what was happening at a specific shift.

**Comment by zuchka** [comment]
Hi @pwSteve,

thanks for making this feature request. Sounds like a useful addition for IoT / Industrial applications.

**Comment by zuchka** [comment]
related to https://github.com/grafana/grafana/issues/300

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
